hadoop-ai/hadoop-build/build.sh

```diff
@@ -31,9 +31,11 @@ git checkout branch-2.9.0
 
 cp /hadoop-2.9.0.gpu-port.patch /hadoop
 cp /HDFS-13773.patch /hadoop
+cp /docker-executor.patch /hadoop
```

Comment:
>/docker-executor.patch [](start = 3, length = 22)

This file is under hadoop/hadoop-ai/hadoop-build.   this is node current folder ?

Reply:
this file is copy to root folder in dockerfile, Need add some comment here.

---
In reply to: [214570065](https://github.com/Microsoft/pai/pull/1247#discussion_r214570065) [](ancestors = 214570065)